src/App.tsx

```diff
@@ -1,25 +1,117 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { TodoWithUser } from './types/TodoType';
+
+function findUsers(userId: number) {
+  return usersFromServer.find((user) => user.id === userId) || null;
+}
+
+const dataList = todosFromServer.map((todo) => {
```

Comment:
```suggestion
const todosWithUsers = todosFromServer.map((todo) => {
```
Would be great to use descriptive meaningful names